ArduCopter/flight_mode.cpp

```diff
@@ -144,17 +142,17 @@ bool Copter::set_mode(control_mode_t mode, mode_reason_t reason)
     control_mode_reason = reason;
     DataFlash.Log_Write_Mode(control_mode);
 
-        adsb.set_is_auto_mode((mode == AUTO) || (mode == RTL) || (mode == GUIDED));
+    adsb.set_is_auto_mode((mode == AUTO) || (mode == RTL) || (mode == GUIDED));
```

Comment:
is this the same as `flightmode->is_auto_pilot()`.

Reply:
Yes, it does appear to be the same with the exception that SmartRTL also returns true for "is_auto_pilot".. but SmartRTL should probably be in that list of modes in any case.  I'll claim, "out of scope" on this request though 'cuz this PR is just for renaming.